setup.py

```diff
@@ -1,24 +1,23 @@
-import os
-import re
+import json
 from setuptools import find_packages, setup
 
 
-def get_version(package):
+def get_version():
     """
-    Return package version as listed in `__version__` in `version.py`.
+    Return package version as in package.json.
     """
-    init_py = open(os.path.join(package, 'version.py')).read()
-    return re.search("__version__ = ['\"]([^'\"]+)['\"]", init_py).group(1)
+    return json.load(open('package.json', 'r'))['version']
```

Comment:
This is a cool idea. However, you need to add a ``MAINFEST.in`` file to make sure ``package.json`` gets distributed in the sdist. I did that [here](https://github.com/mozilla/mozpay-py/blob/master/MANIFEST.in) once to distribute a ``requirements.txt`` file.